boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -63,24 +63,21 @@ clean:
 isclean:
 	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." >&2 && exit 1)
 
-.PHONY: build
-build: isclean
+.PHONY: docker-build
+docker-build: isclean
 	${CONTAINER_ENGINE} build . -f $(OPERATOR_DOCKERFILE) -t $(OPERATOR_IMAGE_URI)
 	${CONTAINER_ENGINE} tag $(OPERATOR_IMAGE_URI) $(OPERATOR_IMAGE_URI_LATEST)
 
-.PHONY: push
-push:
+.PHONY: docker-push
+docker-push:
 	${CONTAINER_ENGINE} push $(OPERATOR_IMAGE_URI)
 	${CONTAINER_ENGINE} push $(OPERATOR_IMAGE_URI_LATEST)
 
-# These names are used by the app-sre pipeline targets
-.PHONY: docker-build
-docker-build: build
-.PHONY: docker-push
-docker-push: push
+.PHONY: push
+push: docker-push
 
 .PHONY: gocheck
```

Comment:
Can we standardize on a naming convention here. I'd prefer something like `<noun>-<verb>` in kebab-case for consistency. 
Can we change all of the follow to `go-<verb>` please?

Reply:
Done via #51. If you have suggestions for how to rename the targets added in this PR to be `noun-verb`ish, I'll update this one.